### PR TITLE
'Re-memes' certain languages.

### PR DESCRIPTION
**Pull request by theselfish**
## About The Pull Request
Host request.
These were removed for dumb reasons. They are now back.
## How This Contributes To The Skyrat Roleplay Experience

I don't know. We are still not Skyrat.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  I can't test this. Why? I'm clueless, but whenever I use your build cmd/bat it throws an error I have quite literally never seen before.
image

</details>
image

## Changelog

:cl: Cursor
add: Pirate, neko and Common are back.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/customization/modules/language/nekomimetic.dm [hunk 1]
```diff
@@ -1,3 +1,6 @@
-// Nekomimetic, my beloved
+/*/ Nekomimetic, my beloved
 /datum/language/nekomimetic
-	secret = TRUE
+*/
+	secret = FALSE //BUBBERSTATION EDIT ORIGINAL secret = FALSE
```
Comment (Cyprex) [inline modular_skyrat/modules/customization/modules/language/nekomimetic.dm:4]: This won't work, this needs to be scoped

File: modular_skyrat/modules/customization/modules/language/piratespeak.dm [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
 	name = "Pirate Gutter"
 	desc = "The language of space pirates. Sounds similar to gutter, though intentionally filled with odd slang to make deciphering hard."
 	space_chance = 60
 	syllables = list(
 		"yaye", "alya", "chirka", "tikat", "bazar", "oi", "yo", "meit", "grok", "ken", "berk", "cohon", "pendeo", "mark",
 		"ho", "take", "yeer", "cur", "yoro", "armn", "tirta", "ren", "wap", "wop", "ran", "cro", "woh"
 	)
-	secret = TRUE
+	*/
+	secret = FALSE //BUBBERSTATION EDIT ORIGINAL secret = TRUE
```
Comment (Cyprex) [inline modular_skyrat/modules/customization/modules/language/piratespeak.dm:10]: same as above

**Review by Cyprex — CHANGES_REQUESTED** [review]
see above

**Comment by Sweettoothart** [comment]
Why is yangu being removed? Let Asian characters not have to speak mew meow language.

**Comment by nevimer** [comment]
stale

**Pull request closed without merging**